### PR TITLE
Submitting py-lru-cache

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

**Comment by ericmjl** [comment]
@conda-forge/help-python I'm not sure why this one is failing on AppVeyor. Would you be kind enough to look into it please? Everything else passes on CircleCI and Travis.

**Comment by volpatto** [comment]
@ericmjl, everything is fine, just a network connection. I ran it again and now passed. See [here](https://github.com/conda-forge/conda-forge-ci-setup-feedstock/issues/43).

File: recipes/py_lru_cache/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - python
+
+test:
+  imports:
+    - lru
+
+about:
+  home: https://github.com/stucchio/Python-LRU-cache
+  license: BSD
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
```
Comment (chrisburr) [inline recipes/py_lru_cache/meta.yaml]:
```suggestion
```

File: recipes/py_lru_cache/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - lru
+
+about:
+  home: https://github.com/stucchio/Python-LRU-cache
+  license: BSD
```
Comment (chrisburr) [inline recipes/py_lru_cache/meta.yaml]:
```suggestion
  license: BSD-3-Clause
```

**Comment by ericmjl** [comment]
Thanks @chrisburr!

**Pull request merged**